src/main/java/cz/codecamp/logger/LoggerInterface.java

```diff
@@ -1,5 +1,17 @@
 package cz.codecamp.logger;
 
-public interface LoggerInterface {
-    void log(LogLevelEnum level, String message);
+import java.io.Closeable;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public interface LoggerInterface extends Closeable {
+    void log( LogLevelEnum level, String message );
+
+    void setFormatter( FormatterInterface formatter );
```

Comment:
Nejsem si jistý, že by tyhle settery měly být součástí `LoggerInterface` - souvisí s implementací, nikoli s intentem uživatele interface - a to, že chci něco zalogovat.